src/account/ProfileSubmenu.test.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
```

Comment:
These tests were largely ported over from the existing tests on the ProfileMenu screen.